ops/generate-subgraph.js

```diff
@@ -66,9 +66,13 @@ function combineFragments(fragments, isTemplate, addresses, missingAddresses) {
       eventHandlers = yamlLoad.eventHandlers;
       entities = yamlLoad.entities;
       abis = (yamlLoad.abis || [contract]).map(contractName => {
+        let correctedVersion = version;
+        if(contractName == 'TokenTrade' && parseInt(version.slice(-1)[0]) <= 3) {
```

Comment:
This is because when this script runs, it generates in the `subgraph.yaml` the following entry in the `abis` of controller:
```
path/ops/../abis/0.1.2-rc.0/TokenTrade.json
```
Causing an error, because the TokenTrade is after the version `.3` - If there's a better solution please, let us know @ben-kaufman